src/components/tasks/card/index.tsx

```diff
@@ -33,18 +31,30 @@ const Card: FC<Props> = ({ content }) => {
   const localEndsOn = new Date(parseInt(endsOn, 10) * 1000);
   const fromNowEndsOn = moment(localEndsOn).fromNow();
 
-  const statusFontColor = status === ACTIVE || ASSIGNED || COMPLETED || PENDING ? '#00a337' : '#f83535';
+  const statusFontColor = status === ACTIVE || ASSIGNED || COMPLETED || PENDING
+    ? '#00a337'
+    : '#f83535';
 
   const iconHeight = '25px';
   const iconWidth = '25px';
 
+  const cardClassNames = [classNames.card];
+  const currentDate = new Date();
+  if (status !== COMPLETED) {
+    if (+localEndsOn - (+currentDate) <= 0) {
+      cardClassNames.push(classNames.overdueTask);
+    }
+  }
   return (
-    <div className={classNames.card}>
+    <div className={cardClassNames.join(' ')} id="wrapper-div">
       <div className={classNames.cardItems}>
         <span className={classNames.cardTitle}>{title}</span>
         <span>
           <span className={classNames.cardSpecialFont}>Status:</span>
-          <span className={classNames.cardStatusFont} style={{ color: statusFontColor }}>
+          <span
+            className={classNames.cardStatusFont}
+            style={{ color: statusFontColor }}
+          >
```

Comment:
Can we revert the indentation change?

Reply:
For sure 😁 I'll disable prettier for the entire file and revert all the changes

Reply:
reverted. please check.